### PR TITLE
Feature/90 hiring staff need a way to get to back to their landing page

### PR DESCRIPTION
**Pull request by gazaston**
Addresses https://trello.com/c/hBO26Tox/90-hiring-staff-need-a-way-to-get-to-back-to-their-landing-page

Before:
screen shot 2018-04-24 at 12 24 51

After:
screen shot 2018-04-24 at 12 24 58

### CODE REVIEW

**Comment by gazaston** [comment]
[Hiring staff need a way to get to back to their landing page](https://trello.com/c/hBO26Tox/90-hiring-staff-need-a-way-to-get-to-back-to-their-landing-page)

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,5 +12,6 @@
 //
 //= require rails-ujs
 //= require jquery
+//= require jquery_ujs
```
Comment (tahb) [inline app/assets/javascripts/application.js]: Was this meant to be in this PR?

Comment (gazaston) [reply app/assets/javascripts/application.js]: Nope. Removed!

**Comment by gazaston** [comment]
[Improved dashboard link for hiring staff](https://trello.com/c/EYiZC18m/97-improved-dashboard-link-for-hiring-staff)

File: spec/features/hiring_staff_can_visit_the_public_listings_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature 'School viewing public listings' do
+  let(:school) { create(:school) }
+  before(:each) do
+    stub_hiring_staff_auth(urn: school.urn)
+  end
+
+  scenario 'A signed in school should see a link back to their own dashboard when viewing public listings' do
+    visit school_path(school)
```
Comment (tahb) [inline spec/features/hiring_staff_can_visit_the_public_listings_spec.rb]: This test would be more reliable if it followed what the user would do in their browser. They'd start on the root path and click about

1. Then click the link to sign in as a school
2. Then click the 'School Jobs' title to go back to the public side
3. Then finally click the new My school link
4. And check the page contains something correct.

**Pull request merged**